tests/Twig/Extension/GithubExtensionTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Tests\Twig\Extension;
+
+use App\Twig\Extension\GithubExtension;
+use PHPUnit\Framework\TestCase;
+
+class GithubExtensionTest extends TestCase
+{
+    /**
+     * @var GithubExtension
+     */
+    private $githubExtension;
```

Comment:
Typed property instead of phpdoc?

Reply:
Introduced in 7.4. And we're using 7.3

Reply:
Yes I read in the other PR, but we should switch to 7.4